### DISCUSSION TITLE
My mobile phone is being hacked. Please help!!!

### DISCUSSION THREAD

**Discussion by leftyprs529** [community/community#30195] [Other Feature Feedback, Questions, & Ideas]
My phone is being hacked and sent out to many people who can access everything on my device. How do I begin to stop this attack, and get my privacy back?

**Comment by queenofcorgis** [comment]
Hi @leftyprs529,
Thanks for being a part of the GitHub Community! We're sorry to hear you're experiencing this. 

I wanted to jump in and mention that if you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on your cell phone's carrier or operating system. It's definitely possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.
